app/src/main/java/com/example/neighborly/FeedFragment.java

```diff
@@ -31,7 +31,8 @@
 
 public class FeedFragment extends Fragment {
     private final FirebaseDatabase database = FirebaseDatabase.getInstance();
-    private static final String introText = "Hi %1s, we found neighbors that have the item you were looking for!";
+    private static final String foundIntroText = "Hi %1s, we found neighbors that have the item you were looking for!";
+    private static final String notFoundIntroText = "Hi %1s, we haven't found neighbors that have the item you were looking for. :(";
```

Comment:
אני חושבת ש
we didn't find
זה יותר זורם

אגב מה זה %1s?
לא הכרתי את הסינטקס :)